app/models/other_party.rb

```diff
@@ -1,5 +1,4 @@
 class OtherParty < Person
-  def full_address
-    address
-  end
+  store_accessor :address_data, :address_line_1, :address_line_2,
```

Comment:
Let's move now this accessor (in applicant, respondent and other_party) to the super class `Person` so everybody inherits it (I know we don't use addresses for children/minors but it is ok to have the accessor, as they also inherits the rest of the table attributes regardless of being used or not in form objects).